browser/src/App.js

```diff
@@ -93,7 +93,21 @@ const navLinks = [
   },
 ];
 
+const fetchContentCollection = () => {
+  const requestContentCollection = new Request('/content', {
+    method: 'GET',
+  });
+
+  fetch(requestContentCollection)
+    .then(response => {
+      response.json();
+      console.log(response);
+    })
+    .then(data => console.log('Success:', data))
```

Comment:
For this to work, you'll have to return something from the previous "then" function body.  Try to make these functions as small as possible.

```js
    .then(response =>  response.json())
    .then(console.log)
    .catch(console.error)
```